generic_tracers/cobalt_glbl.F90

```diff
@@ -0,0 +1,1531 @@
+!> COBALT_glbl module consists of core parameters and vars 
+!! to be used by generic COBALT related modules
+!<----------------------------------------------------------------
+module cobalt_glbl
+  use field_manager_mod, only: fm_string_len, fm_path_name_len      
+  implicit none
+  public  
```

Comment:
The MOM6 convention would be to use 
```
implicit none ; private 
public [list of public types/routines in this module]
```
here.  I think this would impact the parameters listed from lines ~11-54 which are not within a type. But I think each one could be specified as public.
I personally like this format because it is a quick way to find out what types/routines are within a file when you first open it.

Reply:
Make the module private now and specify individual variables as public.